src/Service.Host/client/src/components/requisitions/components/Document1.js

```diff
@@ -103,12 +105,13 @@ function Document1({
                 onSelect({
                     partNumber: line.articleNumber,
                     partDescription: line.description,
-                    document1Line
+                    document1Line,
+                    getReqToReverse: isReverseTransaction
```

Comment:
if getReqToReverse is just being used to check if something has been populated should we just stick something from the PO / CO / WO etc into it rather than adding isReverseTransaction to the watch fields in all the useEffects ?

Reply:
its being used to indicate that the document being reversed is for reversal so the handler can bring up the pick a req popup.  previously it relied on the existence of an orderNumber which only existed for a WO and now has to apply for a few other functions - what sort of thing were you thinking of?